docs/api/microcosm.md

```diff
@@ -104,20 +104,38 @@ manage the provided `key`. Whenever this domain responds to an action,
 it will be provided the current state for that particular key.
 
 `options` passed as the third argument are sent into a domain's
-`setup` method and, if using a class, the constructor.
+`setup` method and, if using a class, the constructor is instantiated
+with the provided options and associated repo.
 
```

Comment:
There is no reason. I'll add an issue for this. We also need to better describe how you can create domains and effects (simple objects vs classes)